ravenframework/Models/PostProcessors/Validations/Representativity.py

```diff
@@ -0,0 +1,489 @@
+      UncertaintyinCorrectedParameters: $parTildeVar \in \mathbb{R}^{P \times P}$
+      CorrectedTargets: $TarTilde \in \mathbb{R}^{F}$
+      UncertaintyinCorrectedTargets:$TarTildeVar \in \mathbb{R}^{F \times F}$
+      ExactUncertaintyinCorrectedTargets:$TarTildeVar \in \mathbb{R}^{F \times F}$
+    """
+    outs = {}
+    for i,param in enumerate(self.featureParameters):
+      name4 = "CorrectedParameters_{}".format(param.split("|")[-1])
+      outs[name4] = correctedParameters[i]
+      for j, param2 in enumerate(self.featureParameters):
+        if param == param2:
+          name5 = "VarianceInCorrectedParameters_{}".format(param.split("|")[-1])
+          outs[name5] = correctedParametersCovariance[i,i]
+        else:
+          name6 = "CovarianceInCorrectedParameters_{}_{}".format(param.split("|")[-1],param2.split("|")[-1])
+          outs[name6] = correctedParametersCovariance[i,j]
+
+    for i,targ in enumerate(self.targets):
+      name3 = "CorrectedTargets_{}".format(targ.split("|")[-1])
+      outs[name3] = correctedTargets[i]
+      for j,feat in enumerate(self.features):
+        name1 = "BiasFactor_Mock{}_Tar{}".format(feat.split("|")[-1], targ.split("|")[-1])
+        name2 = "ExactBiasFactor_Mock{}_Tar{}".format(feat.split("|")[-1], targ.split("|")[-1])
+        outs[name1] = r[i,j]
+        outs[name2] = rExact[i,j]
+      for k,tar in enumerate(self.targets):
+        if k == i:
+          name3 = "CorrectedVar_Tar{}".format(tar.split("|")[-1])
+          name4 = "ExactCorrectedVar_Tar{}".format(tar.split("|")[-1])
+        else:
+          name3 = "CorrectedCov_Tar{}_Tar{}".format(targ.split("|")[-1], tar.split("|")[-1])
+          name4 = "ExactCorrectedCov_Tar{}_Tar{}".format(targ.split("|")[-1], tar.split("|")[-1])
+        outs[name3] = UtarVarTilde[i,k]
+        outs[name4] = UtarVarTildeExact[i,k]
+    return outs
+
+  def _generateSensitivityMatrix(self, outputs, inputs, sensDict, datasets, normalize=True):
+    """
+      Reconstruct sensitivity matrix from the Basic Statistic calculation
+      @ In, inputs, list, list of input variables
+      @ In, outputs, list, list of output variables
+      @ In, sensDict, dict, dictionary contains the sensitivities
+      @ Out, sensMatr, numpy.array, 2-D array of the reconstructed sensitivity matrix
+    """
+    sensMatr = np.zeros((len(outputs), len(inputs)))
+    inputVars = [x.split("|")[-1] for x in inputs]
+    outputVars = [x.split("|")[-1] for x in outputs]
+    for i, outVar in enumerate(outputVars):
+      for j, inpVar in enumerate(inputVars):
+        senName = "{}_{}_{}".format(self.senPrefix, outVar, inpVar)
+        # Assume static data (PointSets are provided as input)
+        if not normalize:
+          sensMatr[i, j] = sensDict[senName][0]
+        else:
+          sensMatr[i, j] = sensDict[senName][0]* datasets[inpVar].meanValue / datasets[outVar].meanValue
+    return sensMatr
+
+  def _getDataFromDatasets(self, datasets, var, names=None):
```

Comment:
This method is in ValidationBase.py class, I think you can merge them.

Reply:
I cannot find it in ValidationBase

Reply:
![image](https://github.com/idaholab/raven/assets/7321071/d33008c4-ae6f-44f3-9af7-20ade60c2de3)
If you can not find it, I guess that you need to merge devel.

Reply:
This is from Dict while the other is from Dataset. Do you want me to move it to ValidationBase?

Reply:
@wangcj05, I discovered that I am no longer using this function. I will remove it.OK?